javascript/scripts/build.mjs

```diff
@@ -104,10 +105,28 @@ function buildWasm(outputDir) {
         cwd: __dirname,
       },
     )
+
+    // add /* @vite-ignore */ comment to URL imports for wasm
+    // prevents vite from bundling multiple wasm files
+    // see https://github.com/automerge/automerge/issues/1037
```

Comment:
As well as referencing the issue I think it would be good to write in some prose about the silly things vite is doing here which makes it necessary, as well as the fact that the vite comment is an undocumented thing we are taking advantage of.

It's good to reference issues, but you never know when GitHub will be down.